packages/xrpl/test/client/getFeeXrp.test.ts

```diff
@@ -2,29 +2,40 @@ import { assert } from 'chai'
 
 import getFeeXrp from '../../src/sugar/getFeeXrp'
 import rippled from '../fixtures/rippled'
-import { setupClient, teardownClient } from '../setupClient'
+import {
+  setupClient,
+  teardownClient,
+  type XrplTestContext,
+} from '../setupClient'
 
 describe('getFeeXrp', function () {
-  beforeEach(setupClient)
-  afterEach(teardownClient)
+  let testContext: XrplTestContext
+
+  beforeEach(async () => {
```

Comment:
ditto

Reply:
This is a new new function, so it's fine to leave it